packages/patternfly-3/patternfly-react/src/components/LoadingState/LoadingState.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Spinner } from '../Spinner';
+
+class LoadingState extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      render: false
+    };
+  }
+
+  componentDidMount() {
+    setTimeout(() => {
+      this.setState({ render: true });
+    }, this.props.timeout);
+  }
+
+  render() {
+    const { loading, loadingText, children } = this.props;
+    const spinner = (
+      <div className="pf-loading-state">
+        <Spinner loading={loading} size="lg" />
```

Comment:
we should allow a prop to set the size and add the appropriate class to the pf-loading-state as well as set the size on the Spinner.